src/backend/CourseDb.js

```diff
@@ -76,6 +76,19 @@ class CourseDb {
         return doc.students;
     }
 
+    async getCourses(teacherId) {
```

Comment:
What I mentioned above: the function name should reflect the functionality of getting courses for a teacher id.

Reply:
see above